server/package.json

```diff
@@ -7,7 +7,7 @@
     "server:dev": "nodemon index.js",
     "test": "jest --coverage",
     "test:watch": "jest --watch",
-    "lint": "eslint --fix '**/*.js'",
+    "lint": "eslint --fix \"**/*.js\"",
```

Comment:
For some reason lint was complaining about the ' ' quotes, so I replaced them with escaped double quotes

Reply:
Yeah I think this is a weird linux / windows thing. I tested this part with your updates and it still works, so this part is good!